web/server/codechecker_server/migrations/config/env.py

```diff
@@ -1,7 +1,7 @@
-from __future__ import with_statement
-from __future__ import print_function
-from __future__ import division
-from __future__ import absolute_import
+
+
+
+
```

Comment:
Do we want to add empty lines here rather than removing the lines as well? And other places.

Reply:
I did not run the source code formatter for the schema migration related scripts not to mess up something that is why they were left there, I will remove them.